Dependency issue with fastboot-app-server

I am getting the following error when attempting to use an ember app that uses `ember-fetch` with `fastboot-app-server`

```
Error: Cannot find module 'abortcontroller-polyfill/dist/cjs-ponyfill'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.require (/Users/user/node_modules/fastboot/src/ember-app.js:130:18)
    at s.callback (/Users/user/fb-test/dist/ember-fetch/fastboot-fetch-ee5cef26e0470d371ec08fef9ba4a5b8.js:1:55)
    at s.exports (/Users/user/fb-test/dist/assets/vendor-12033a7caf9c9a2eebca95e833f776cb.js:19:21)
    at s._reify (/Users/user/fb-test/dist/assets/vendor-12033a7caf9c9a2eebca95e833f776cb.js:22:35)
    at s.reify (/Users/user/fb-test/dist/assets/vendor-12033a7caf9c9a2eebca95e833f776cb.js:21:23)
    at s.exports (/Users/user/fb-test/dist/assets/vendor-12033a7caf9c9a2eebca95e833f776cb.js:18:84)
```

I do not get this error when visiting the same page when serving the app with `ember s`

Here is a very simple app that will recreate this error when using the `fastboot-app-server` defaults:

https://github.com/Mciocca/fastboot-test

A co-worker was also able to recreate this error. He followed the tutorial on the ember fastboot site.

--- a/package.json
+++ b/package.json
@@ -66,6 +66,7 @@
     "configPath": "tests/dummy/config",
     "fastbootDependencies": [
       "node-fetch",
+      "abortcontroller-polyfill",
       "abortcontroller-polyfill/dist/cjs-ponyfill"
     ]
   }

